app/src/main/java/com/example/mycalculator/mvp/CalculatorModel.java

```diff
@@ -0,0 +1,15 @@
+package com.example.mycalculator.mvp;
```

Comment:
yo can create a package called model, and put this class inside

Reply:
Done. Thx!